java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosure1.ql

```diff
@@ -0,0 +1,72 @@
+/**
+ * @name Temporary Directory Local information disclosure
+ * @description Detect local information disclosure via the java temporary directory
+ * @kind problem
+ * @problem.severity warning
+ * @precision very-high
+ * @id java/local-information-disclosure
```

Comment:
This `id` is rather generic. Could you make it a bit more specific to this query?

Reply:
Fixed, I think